src/core/Vector.hpp

```diff
@@ -164,6 +164,7 @@ template <size_t n, typename Scalar> class Vector {
 // Useful typedefs
 
 typedef Vector<3, double> Vector3d;
+typedef Vector<3, int> Vector3dInt;
```

Comment:
That should be called `Vector3i`. Please also use `using` over `typedef`.

Reply:
```using``` and ```typedef``` are equivalent and i adopted to the existing usage.